include/nlohmann/detail/input/json_sax.hpp

```diff
@@ -298,6 +325,62 @@ class json_sax_dom_parser
     }
 
   private:
+
+    void set_start_end_pos_for_handle_value(BasicJsonType& v)
```

Comment:
Didn't understand what handle means here

Reply:
I meant it as for the function `handle_value()`, which should be the only place where this helper is called from